src/core/storybook/serviceUrlArgs.ts

```diff
@@ -14,7 +14,7 @@ export default {
   },
   [serviceUrlKeys.dplCms]: {
     name: "Base url for the DPL CMS API",
-    defaultValue: "https://dpl-cms.docker",
+    defaultValue: process.env.CMS_BASEURL ?? "https://dpl-cms.docker",
```

Comment:
Is this correct? 

I wonder why there is a fallback for an .env variable, which defaults to what i assume you are supposed to set in your .env. 

Also in the .env we have another url than `https://dpl-cms.docker`

Not saying anything is necessarily wrong here, just wondering.

Reply:
This is the way we handle working with WireMock. We will only add the `CMS_BASEURL` (via environment variables) if we have to fetch against WireMock.